src/blocks/file-blocks/code/index.tsx

```diff
@@ -89,14 +82,35 @@ export default function (props: FileBlockProps) {
     onRequestUpdateContent,
   } = props;
 
-  doc = content;
   const editorRef = React.useRef<HTMLDivElement>(null);
 
+  const viewRef = React.useMemo<{ current: EditorView | null }>(
+    () => ({ current: null }),
+    []
+  );
```

Comment:
this is essentially `useRef` but the type of actual `useRef` makes `current` read-only.

Reply:
oh, interesting!